### DISCUSSION TITLE
Fails to build for 7520 due to missing file on download mirror

### DISCUSSION THREAD

**Discussion by HashfoxGmbH** [Freetz-NG/freetz-ng#778] [Allgemein] [answered]
Hello!

Building the current master branch of Freetz currently fails for model 7520 (and possibly others) due to file 'kernel_4.4.271-7530_07.50-ADv1.patch.xz' with sha256 '3c73cfdcfec51be6ce9549a325899ded9a5b3586f6d3619b9211453d28bbc434' not being available for download.

This file was introduced via commit c71f8e80092787e7c9af5b1db673a83a241d3f0e

Any help appreciated! =)

The actual error:

downloading ... tools/freetz_download dl kernel_4.4.271-7530_07.50-ADv1.patch.xz @MIRROR/ 3c73cfdcfec51be6ce9549a325899ded9a5b3586f6d3619b9211453d28bbc434
..
--2023-04-04 15:02:11--  https://raw.githubusercontent.com/Freetz-NG/dl-mirror/master/kernel_4.4.271-7530_07.50-ADv1.patch.xz
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.109.133, 185.199.110.133, 185.199.108.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.109.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-04-04 15:02:11 ERROR 404: Not Found.

Download failed - "https://raw.githubusercontent.com/Freetz-NG/dl-mirror/master/kernel_4.4.271-7530_07.50-ADv1.patch.xz"  ->  error code 8

**Comment by fda77** [comment, marked as answer]
I've uploaded it to the github mirror: https://github.com/Freetz-NG/dl-mirror/commit/907ba49 . On the 1st mirror your network seems to be blocked ( i cant change it)

**Comment by HashfoxGmbH** [comment]
Thank you for the fast reaction, the build works again! =)
